src/main/webapp/resources/js/pages/analysis/components/outputs/OutputFilePreview.jsx

```diff
@@ -45,6 +46,14 @@ export default function OutputFilePreview() {
     }
   }, []);
 
+  function getMenu () {
```

Comment:
As I was going through taking screen shots I came across the inability to download individual files that don't have a preview available. Since it wasn't much of a change I just put it in here